mne/time_frequency/tfr.py

```diff
@@ -742,9 +745,26 @@ def plot_topo(self, picks=None, baseline=None, mode='mean', tmin=None,
         border : str
             matplotlib borders style to be used for each sensor plot.
         fig_facecolor : str | obj
-            The figure face color. Defaults to black.
-        font_color: str | obj
-            The color of tick labels in the colorbar. Defaults to white.
+            The figure face color. Defaults to black. If "black", spines are
+            white (else they are black).
+        font_color : str | obj
+            The color of text in the colorbar and title. Defaults to
+            spine_color (which is automatically set based on fig_facecolor
+            if None).
+        external_legend : bool | float
```

Comment:
As I've said before, I wouldn't allow the user to scale the legend arbitrarily. Let's try to make it work without the user having to scale it.